Registry/RegistryParse.py

```diff
@@ -279,6 +294,49 @@ def minor_version(self):
         """
         return self.unpack_dword(0x18)
 
+    def clustering_factor(self):
+        """
+        Get the clustering factor.
+        """
+        return self.unpack_dword(0x2C)
+
+    def file_type(self):
+        """
+        Get the file type.
+        """
+        return self.unpack_dword(0x1C)
+
+    def is_primary_file(self):
+        """
+        Check if this REGF block belongs to a primary (normal) file.
+        """
+        return self.file_type() == 0
```

Comment:
we should use an `enum` to make these constants less "magic". example using `enum34` is here: https://github.com/williballenthin/python-registry/blob/master/Registry/Registry.py#L43